plugins/catalog-backend/src/catalog/DatabaseLocationsCatalog.ts

```diff
@@ -42,13 +47,36 @@ export class DatabaseLocationsCatalog implements LocationsCatalog {
     await this.database.removeLocation(id);
   }
 
-  async locations(): Promise<Location[]> {
+  async locations(): Promise<LocationEnvelope[]> {
```

Comment:
can we call this `LocationResponse`?

Reply:
- [x] `LocationEnvelope` -> `LocationResponse`